tests/examples/gears/shape/radial_progress.lua

```diff
@@ -1,13 +1,37 @@
 --DOC_GEN_IMAGE --DOC_HIDE
 local shape,cr,show = ... --DOC_HIDE
 
+shape.radial_progress(cr, 70, 20, 0)
+-- show(cr, true) --DOC_HIDE
```

Comment:
This script is used to generate images and code usage example at https://awesomewm.org/apidoc/theme_related_libraries/gears.shape.html#radial_progress.

By commenting this line, the generated image will not have this radial_progress widget in. But the usage example code will still have the previous line, which defines the widget. I think this is confusing for someone reading the documentation.

Same apply to next blocks.

Reply:
OK! I get it! I just wanted to add more code coverage to the commit. There is a lot of `if-then-else` tests so that codecov hardly covers all these situations.
I don't think that the documentation needs all of these examples but codecov needs them. Is there another file where I can put them?

Reply:
I don't know how codecov computes the coverage, so please correct me if I'm wrong, but AFAIK, `tests/examples/*/*` shouldn't be part of the coverage status. Theses scripts are run at build time to generate code examples and images used by the documentation.

If you want to test things, you should take a look at [busted](https://olivinelabs.com/busted/) and files from the  [spec/](https://github.com/awesomeWM/awesome/tree/master/spec) directory.

Additionally, script at `tests/test-*.lua` can be used as acceptance tests to check components behave as intended. These tests are launched into a Xephyr session.